### PR TITLE
ZWEB-8304 SL Update zerv-core version

### PR DESCRIPTION
**Pull request by Semen-Lohunov**
**Updates**:

1. Update zerv-core version
2. Update jasmine setup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/z-open/zerv-sync.git"
   },
   "scripts": {
-    "test": "./node_modules/jasmine/bin/jasmine.js",
+    "test": "./node_modules/jasmine/bin/jasmine.js spec/*.spec.js",
```
Comment (Jacob-Friesen) [inline package.json:16]: This will not catch any tests made in folders of the `spec` folder.

**Review by Jacob-Friesen — CHANGES_REQUESTED** [review]
This works as expected except for the jasmine test change.

File: package.json (hunk 1 above)
Comment (Jacob-Friesen) [reply package.json]: Now if I add a test file in a folder, `npm test` just runs that test and not anything in the main spec folder.

Comment (Semen-Lohunov) [reply package.json]: Updated

**Review by Jacob-Friesen — APPROVED** [review]

**Pull request merged**